GitUI/CommandsDialogs/FormClone.cs

```diff
@@ -70,6 +70,31 @@ protected override void OnRuntimeLoad(EventArgs e)
                     _NO_TRANSLATE_To.Text = Module.WorkingDir;
             }
 
+            if (_NO_TRANSLATE_To.Text == _NO_TRANSLATE_From.Text ||
+                _NO_TRANSLATE_To.Text == PathUtil.EnsureTrailingPathSeparator(_NO_TRANSLATE_From.Text))
+            {
+                // Try to be more helpful to the user.
+                // Use the cliboard text as a potential source URL.
+                try
+                {
+                    if (Clipboard.ContainsText(TextDataFormat.Text))
+                    {
+                        string text = Clipboard.GetText(TextDataFormat.Text) ?? string.Empty;
+
+                        // See if it's a valid URL.
+                        string lowerText = text.ToLowerInvariant();
+                        if (lowerText.StartsWith("http") ||
```

Comment:
What do you think about checking if text contains 'git' when it starts from 'http'?
https://jbialobr@bitbucket.org/jbialobr/gitextensions.git
https://github.com/gitextensions/gitextensions.git

Another common URI format is ssh:
ssh://janusz@srvname/path/to/the/repo